Specs/JWPlayerPlugin/3.3.4/JWPlayerPlugin.podspec

```diff
@@ -0,0 +1,31 @@
+Pod::Spec.new do |s|
+
+  s.name             = "JWPlayerPlugin"
+  s.version          = '3.3.4'
+  s.summary          = 'JWPlayer Player plugin implementation.'
+  s.description      = 'An implementation for JWPlayer as a Zapp Player Plugin in Objective C.'
+  s.homepage         = "https://github.com/applicaster/zapp-player-plugin-jw"
+  s.license          = 'MIT'
+  s.author           = { "Jesus De Meyer" => "j.demeyer@applicaster.com" }
+  s.source           = { :git => "git@github.com:applicaster/zapp-player-plugin-jw.git", :tag => s.version.to_s }
```

Comment:
We use another name conventions for tag names in monorepos. All new tags should have format like "ios-1.1.1" that also imply changes in pod specs. You can look at [cleeng repo](https://github.com/applicaster/zapp-login-plugin-cleeng/blob/master/Specs/CleengLogin/1.1.0/CleengLogin.podspec) as reference

Reply:
PR updated :)